tests/ci/integration/mariadb_patch/0003-WIP-Use-replace_regex-in-mysql-test-run-to-make-SSL-.patch

```diff
@@ -0,0 +1,28 @@
+From 557b4261a3120b037e2c705ce94453bc63d08823 Mon Sep 17 00:00:00 2001
+From: Otto Kekalainen <ottoke@amazon.com>
+Date: Mon, 28 Nov 2022 21:03:50 -0800
+Subject: [PATCH 3/4] WIP: Use `--replace_regex` in mysql-test-run to make SSL
+ tests pass
+
+Another potential area worthy of upstream contribution.
+
+The new error messages _seem_ to follow uppercase and underscore format.
```

Comment:
Who is different here? Is this 1.0.2 vs AWS-LC **and** 1.1.1? Or is AWS-LC the only different one?

Reply:
This is referring to our error messages which look like this `SSLV3_ALERT_CERTIFICATE_REVOKED`.
OpenSSL's error messages have lower cases and spaces instead: `sslv3 alert certificate revoked`.